misc/review/MidiServiceConfigurationResponseStatusEnum.idl

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+// ============================================================================
+// This is part of the Windows MIDI Services App API and should be used
+// in your Windows application via an official binary distribution.
+// Further information: https://github.com/microsoft/MIDI/
+// ============================================================================
+
+#include "namespace_defines.h"
+#include "midl_defines.h"
+MIDI_IDL_IMPORT
+
+namespace MIDI_MIDL_NAMESPACE
+{
+    MIDI_API_CONTRACT_v1
+    enum MidiServiceConfigurationResponseStatus
+    {
+        Success = 0,
+
+        ErrorTargetNotFound = 404,
+
+        ErrorJsonNullOrEmpty = 600,
+        ErrorProcessingJson = 601,
+
+        ErrorNotImplemented = 2600,
+
+        ErrorOther = 9999,
```

Comment:
How will we add future errors? Suppose we want to add a new error "ErrorWrongColor = 3000". This used to be reported as ErrorOther, but if we change it to report ErrorWrongColor, then people who did `if err == ErrorOther` will now break.